shared/src/store/effects/keymap.ts

```diff
@@ -2,11 +2,13 @@ import { Injectable } from '@angular/core';
 import { Router } from '@angular/router';
 
 import { Actions, Effect } from '@ngrx/effects';
-import { Store } from '@ngrx/store';
+import { Action, Store } from '@ngrx/store';
+import { Observable } from 'rxjs/Observable';
 
 import 'rxjs/add/operator/do';
 import 'rxjs/add/operator/map';
 import 'rxjs/add/operator/withLatestFrom';
+import 'rxjs/observable/of';
```

Comment:
Change to 'rxjs/add/observable/of'.
startWith and switchMap operators also need to be imported.